front/src/app/pages/home/home.component.ts

```diff
@@ -57,4 +70,18 @@ export class HomeComponent implements OnInit {
       console.log(err);
     });
   }
+
+  notify(ev: any) {
+    const percent = ev.left / (this.time * 10);
+    this.color = this.hsl_col_perc(percent);
+  }
+
+  hsl_col_perc(percent: number) {
+    const red = 0;
+    const green = 120;
+    const a = percent / 100;
+    const b = (green - red) * a;
+    const c = b + red;
```

Comment:
replace const a, b and c by 'const c = (((green - red) * percent) / 100) + red;'